js/router.js

```diff
@@ -0,0 +1,4 @@
+page('/', articleController.initIndex);
+page('/about', articleView.about);
```

Comment:
Good. You would potentially send this route to something like an `aboutController`, depending upon the type of resource used to create the view.